style.css

```diff
@@ -1503,3 +1507,435 @@
   font-size: 23px;
   margin-left: 18px;
 }
+
+@media only screen and (min-width: 768px) {
+  .toolbar {
+    width: 100vw;
+  }
+
+  .menu-bar {
+    display: flex;
+    width: fit-content;
+    right: 120px;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .nav-menu {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .nav-menu li {
+    color: #fff;
+    margin-left: 80px;
+  }
+
+  .nav-menu li a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .headline {
+    width: 100vw;
+    height: 609px;
+    flex-direction: row;
+    background-image: url(./imgs/GroupBig.svg);
+  }
```

Comment:
Kindly ensure that the background image does not overlap with the text.

Your design at 768px:
![image](https://user-images.githubusercontent.com/20554583/215263656-716f3d37-5efe-4c05-b13d-bb2f1aa60ad1.png)

Reply:
Done.